protorpc/messages.py

```diff
@@ -1152,7 +1152,6 @@ class Field(six.with_metaclass(_FieldMeta, object)):
   #
   # We add an alias here for backwards compatibility; note that in
   # python3, this attribute will silently be ignored.
-  __metaclass__ = _FieldMeta
```

Comment:
Maybe drop the comment/TODO along the way?

Reply:
Done, thanks.